tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/jdt/LiferayComponentProperties.java

```diff
@@ -0,0 +1,346 @@
+/*******************************************************************************
```

Comment:
we need to add more possible compoment properties.  Look here:

https://github.com/gamerson/liferay-portal/tree/a3a8c16043662f9865837e9c8c473ecd5aed5873/modules/sdk/gradle-templates/src/main/resources/standalone

In all of teh templates we need to search for possible keys here.  

Also we need to go through all of the @ Component in the portal modules and see what are all "possible" keys that should be added.  We should probably write a script that scans java files for those components and the properties.  Please add more keys and then we will be ready to merge this one.